<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DemoBatchSolve</class>
 <widget class="QMainWindow" name="DemoBatchSolve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QSplitter" name="horSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="loadB">
            <property name="text">
             <string>Load</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearImagesB">
            <property name="text">
             <string>Clear All</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteImageB">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QListWidget" name="imagesList">
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="selectIndexes">
            <property name="text">
             <string>Add Index File Directory</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="indexDirectories"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QCheckBox" name="saveImages">
              <property name="text">
               <string>Save Results to:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="outputDirectory"/>
            </item>
            <item>
             <widget class="QPushButton" name="selectOutputDir">
              <property name="text">
               <string>Select</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="verSplitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>336</width>
           <height>68</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="ImageLabel" name="imageDisplay">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_4">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="solveB">
             <property name="text">
              <string>Solve Images</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pauseB">
             <property name="text">
              <string>Pause</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="abortB">
             <property name="text">
              <string>Abort</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="logDisplay"/>
         </item>
         <item>
          <widget class="QProgressBar" name="processProgress">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDemoBatchSolve">
    <property name="title">
     <string>DemoBatchSolve</string>
    </property>
   </widget>
   <addaction name="menuDemoBatchSolve"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageLabel</class>
   <extends>QLabel</extends>
   <header location="global">testerutils/imagelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
